src/plugins/dev-server.js

```diff
@@ -6,7 +6,7 @@ module.exports = (app) => {
 	log.debug("Starting server in development mode");
 
 	const webpack = require("webpack");
-	const webpackConfig = require("../../webpack.config.js");
+	const webpackConfig = require("../../webpack.config.js")(undefined, {mode: "production"});
```

Comment:
mode production?

Reply:
`src/plugins/dev-server.js` does in fact use the production webpack config. This is one commit before the cjs changes i PR'd https://github.com/thelounge/thelounge/blob/5f7acbf9948446289054527ee9385f938a116386/src/plugins/dev-server.js#L9 and browsing the repo at that commit shows there is a webpack.config-test.js but that's not what the dev-server is using.